### ISSUE TITLE
Text using opacity makes it tricky to keep an accessible contrast ratio

### ISSUE THREAD

**Issue by ludoboludo** [Shopify/dawn#1521]
**Describe the current behavior**

Currently we're relying on opacity to differentiate some pieces of information. We're applying a `.75` opacity which can cause the color contrast between the text color and the background to be too low. The contrast ratio should be at `4.5` to be considered accessible. 

We could change it so that it's using full opacity instead but we will need to check where it's been used and find alternative ways to have a visual hierarchy. 
Here are some pros and cons to that switch: 
👍  Some existing and new art directions will benefit aesthetically from all the text being 100% opacity
👍  The way colour works in our themes becomes more predictable
👍  Providing we select colours with enough contrast, we will meet accessibility standards.
👎  Some existing themes benefited from the hierarchy that the different opacity levels gave us, however, we can look at providing this as a setting, either through the colour work, or separately in the future.

**Describe the expected behavior**

**Version information (Dawn, browsers and operating systems)**
- Dawn Version: 4.0.0

**Possible solution**
Remove the opacity in most places and rely on a different visual cue. 

**Additional context/screenshots**
Add any other context about the problem here. If applicable, add screenshots to help explain.

cc: @allylane @nicklepine

**Comment by Stephanie-Chou** [comment]
We are closing this issue. This issue was reported 2 years ago on an old version of Dawn. If this is still a high priority issue today, please re open with updated reproduction steps
